app/controllers/missions_controller.rb

```diff
@@ -15,7 +15,7 @@ def new
   def create
     @mission = Mission.new(missions_params)
     @mission.mentor = current_user
-    @mission.duration = (end_date - start_date)
+    @mission.duration = (@mission.end_date - @mission.start_date)
```

Comment:
`@mission.duration = (@mission.end_date - @mission.start_date).to_i`
=> in order to get an integer